pyinfra/facts/server.py

```diff
@@ -27,31 +30,39 @@ class Hostname(FactBase):
 
 class Os(FactBase):
     '''
-    Returns the OS name according to ``uname``.
+    Returns the OS name according to ``uname`` or ``systeminfo``.
     '''
 
-    command = 'uname -s'
+    # command = 'uname -s'
+    # TODO: cmd or ps
+    # TODO: Need a conditional for which one to use. Any suggestions?
```

Comment:
In short - this isn't possible. I think the best option here is to have a specific set of `windows_*` prefixed facts; `facts/windows.py` eg:

+ `WindowsSystemInfo` / `windows_system_info`
+ `WindowsDate` / `windows_date`.

Reply:
Ok. I took a stab at creating an initial windows.py.

Not sure how to let all-facts know when to run which facts. Any ideas?